libs/platform/src/lib/components/link/link.component.ts

```diff
@@ -144,6 +150,8 @@ export class LinkComponent extends BaseComponent implements OnInit, AfterViewIni
         /* If link linkType===emphasized then make link emphasized type */
         if (this.linkType === VALID_INPUT_TYPES[1]) {
             this.emphasized = true;
+        } else if (this.linkType === VALID_INPUT_TYPES[2]) {
+            this.subtle = true;
         }
```

Comment:
looks like the whole thing could be replaced with just 2 lines
```
this.emphasized = this.linkType === VALID_INPUT_TYPES[1];
this.subtle = this.linkType === VALID_INPUT_TYPES[2];
```

Reply:
fixed